context/app/static/js/components/Search/noHitsComponents/noHitsComponents.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { Alert } from 'js/shared-styles/alerts';
+
+function NoResults() {
+  return <Alert severity="warning">No Results Found. Check your spelling or unselect filters.</Alert>;
```

Comment:
```suggestion
  return <Alert severity="warning">No results found. Check your spelling or unselect filters.</Alert>;
```
It might be in the design, but using Title Caps in a sentence like this seems odd to me.